tests/hook_main.py

```diff
@@ -1977,7 +1977,7 @@ def session_track_get_detail(transaction):
     :return:
     """
     with stash['app'].app_context():
-        session = SessionFactory()
+        session = SessionFactory(track_id=1)
```

Comment:
Why can't we define `track_id` variable defined inside the `SessionFactory` with value 1? Why do we need to pass it as kwarg?

Reply:
Yes, for all of them.

Reply:
Alright. Should I make the seed data for event first?

Reply:
@prabhupant What I meant was that I think there is no need to pass `track_id` variable to the `SessionFactory`.
It is already defined as 1 in the factory here https://github.com/fossasia/open-event-server/blob/c8bfed7bc4ae817a6b775cc81098e1fa6925ffa1/app/factories/session.py#L39

Reply:
@srv-twry i made the changes you asked for, but now the hooks are timing out. please help

Reply:
If the hooks were not getting timed out earlier then revert back to what you were doing before. I'll need some time to investigate why it is timing out but currently not in a position to devote that much time(multiple interviews lined up one after the other). So kindly follow what has been done in other places in the project.